runintegrationtests.sh

```diff
@@ -6,6 +6,9 @@
 set -e
 source toolversions.sh
 
+SCRIPT=$(readlink -f "$0")
+ROOT_DIR=$(dirname "$SCRIPT")
```

Comment:
I've generally used `realpath` for this:

```sh
ROOT_DIR=$(realpath $(dirname $0))
```

... but if `dirname` works, that's fine...

Reply:
I really don't know either, it's just what I'd used in some other scripts.

I did a quick Google Search and it seems they are more or less the same.